src/renderer/components/ft-list-video/ft-list-video.scss

```diff
@@ -27,3 +27,18 @@
 .videoTagLine .videoTag:first-child {
   margin-inline-start: 0;
 }
+
+.channelThumbnail {
+  border-radius: 999px;
+  display: inline-block;
+  max-inline-size: 24px;
+  max-block-size: 24px;
+  margin-block-end: 0 !important;
+  margin-inline-end: 0.5rem;
+}
```

Comment:
I know you are trying to override the value in `.ft-list-item.grid` but I think it's better to avoid using `!important` when possible

```suggestion
.channelThumbnail {
  border-radius: 999px;
  display: inline-block;
  max-inline-size: 24px;
  max-block-size: 24px;
  margin-inline-end: 0.5rem;

  @at-root {
    // Required to override style from ft-list-item
    .ft-list-item.grid & {
      margin-block-end: 0 !important;
    }
  }
}
```

What existing style is for:
![image](https://github.com/user-attachments/assets/b6f4cdc7-e90c-4384-af7c-dc7b0eaed0ba)